docs/src/main/sphinx/connector/query-passthrough-warning.fragment

```diff
@@ -0,0 +1,10 @@
+The native query passed to the underlying data source is required to return a
+table as a result set. This can be a one column table with one row. No
+validation or security checks of these queries are performed by Trino. The
+data source performs these tasks using its own configuration.
+
+You can use passthrough queries to read data; however, queries that update,
```

Comment:
lets change this to 

queries that write data typically can not be used since they do not return a table.